avocado/core/sysinfo.py

```diff
@@ -213,6 +214,45 @@ def stop(self):
         return retcode
 
 
+class JournalctlWatcher(Collectible):
+
+    """
+    Track the content of systemd journal into a compressed file.
+
+    :param logf: Basename of the file where output is logged (optional).
+    """
+
+    def __init__(self, logf=None):
+        if not logf:
+            logf = 'journalctl.gz'
+
```

Comment:
PEP8 did not complain about this empty line, but I am complaining... :smile: It's really minor, but IMHO feels a bit inconsistent.